testomania/src/main/java/com/earth/testomania/technical/domain/repository/QuizRepository.kt

```diff
@@ -2,10 +2,11 @@ package com.earth.testomania.technical.domain.repository
 
 import com.earth.testomania.common.DataState
 import com.earth.testomania.technical.data.source.remote.dto.TechQuizDTO
+import com.earth.testomania.technical.domain.model.QuizCategory
 import kotlinx.coroutines.flow.Flow
 
 interface QuizRepository {
 
-    suspend fun getQuizList(): Flow<DataState<List<TechQuizDTO>>>
+    suspend fun getQuizList(params: QuizCategory): Flow<DataState<List<TechQuizDTO>>>
 
```

Comment:
space ebi

Reply:
I think space here is not a problem and it looks better with spaces